projects/igniteui-angular/src/lib/combo/combo.pipes.ts

```diff
@@ -11,12 +11,11 @@ import { IComboFilteringOptions } from './combo.component';
  * @hidden
  */
 @Pipe({
-    name: 'comboFiltering',
-    pure: false
+    name: 'comboFiltering'
 })
 export class IgxComboFilteringPipe implements PipeTransform {
     public transform(collection: any[], searchValue: any, displayKey: any,
-                    shouldFilter: boolean, filteringOptions: IComboFilteringOptions) {
```

Comment:
You don't need to define the pipeTrigger parameter here. As long as you pass the parameter in the pipe calls, it would trigger.